Git.hub/Client.cs

```diff
@@ -157,10 +158,22 @@ public User GetUser(string userName)
 
             var request = new RestRequest($"/users/{userName}");
 
-            var user = client.Get<User>(request);
+            var user = await client.ExecuteGetTaskAsync<User>(request);
             return user.Data;
         }
 
+        public User GetUser(string userName)
```

Comment:
This method doesn't seem to be used anywhere else.
Please remove it.

Reply:
Hi @RussKie ,

Thanks for the comment. I was afraid to break backward compatibility by removing this method. If you feel it's not a problem I will do so.

Reply:
Removed.